UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/appsettings.Development.json

```diff
@@ -1,22 +1,19 @@
 {
-  "Logging": {
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft.AspNetCore": "Warning"
-    }
-  },
-  "FileShareService": {
-    "BaseUrl": "",
-    "BusinessUnit": "",
-    "ProductType": "",
-    "FssClientId": "",
-    "PageSize": "100",
-    "Start": "0"
-  },
+    "Logging": {
+        "LogLevel": {
+            "Default": "Information",
+            "Microsoft.AspNetCore": "Warning"
+        }
+    },
+    "FileShareService": {
+        "BaseUrl": "",
+        "BusinessUnit": "",
+        "ProductType": "",
+        "FssClientId": "",
+        "PageSize": "100",
+        "Start": "0"
+    },
     "RadioNavigationalWarningsContext": {
         "ConnectionString": ""
-    },
-  "RadioNavigationalWarningConfiguration": {
-    "AdminListRecordPerPage": 20
-  }
+    }
```

Comment:
Please revert this file as RadioNavigationalWarningConfiguration is removed which is not a relevant change.